docs/data/product/dea-surface-reflectance-landsat-7-etm/_data.yaml

```
@@ -15,8 +15,10 @@ spatial_data_type: Raster
 time_span:
   start: 28/05/1999
   end: 06/04/2022
-update_frequency: Daily
-next_update: null
+data_updates:
+  frequency: DAILY
+  activity: ONGOING
```

Landsat 7 no longer collecting data (at least not for DEA)
```suggestion
  activity: NO_UPDATES
```

---

@robbibt Which of these would make more sense:

```
   "NO_UPDATES": "No further updates",
   "DECOMMISSIONED": "Decommissioned and no further updates",
```

---

Plain "Decomissioned" is nice; I think you can probably drop "and no further updates" as it's a bit tautological.

Personally, I kind of like the pairing of "No planned updates" and "Decommisioned" - the first one gives the impression that the product is still valid not but we don't currently intend to modify it (but still gives us the option to if we change our minds). The second one is more final - the product is shut down for good. @CEKrause?

---

Or maybe "No further updates" is nice and simple and generic and covers both those cases. Tricky!